src/v2/components/FireListMap.jsx

```diff
@@ -231,6 +245,30 @@ export default function FireMap(props) {
 
 // -----------------------------------------------------------------------------
 
+function addPolygon(fire, polygons, greatPolygon) {
+  const {L} = window
+  const p = getBestPolygon(fire)
+
+  // NOTE: Points passed when creating a polygon shouldn’t have a last
+  // point equal to the first one. It’s better to filter out such points.
+  // https://leafletjs.com/reference.html#polygon
+  const vertices = p.slice(0, p.length - 1)
+
+  greatPolygon.splice(greatPolygon.length, 0, ...vertices)
+
+  polygons.push(L.polygon(vertices, Props.PRIMARY_POLYGON))
+}
+
+function getBestPolygon(fire) {
+  try {
+    return findPrimaryPolygon(fire)
+  } catch (error) {
+    println(error)
+    println('Falling back to fire.polygon')
+    return fire.polygon
+  }
```

Comment:
Unnecessary once you switch findPrimary to return last polygon

Reply:
Done